src/main/java/core/basesyntax/AreaOfFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaOfFigure {
```

Comment:
read about [Java Naming Conventions](https://howtodoinjava.com/java/basics/java-naming-conventions/)
```suggestion
public interface AreaCalculator {
```